feeds/us-ny.json

```diff
@@ -36,6 +36,46 @@
       "type": "transitland-atlas",
       "transitland-atlas-id": "f-dr5r-nyctsubway"
     },
+    {
+      "name": "MTA_NYCSubway",
```

Comment:
The import script fails on these names.
I think in other cases we use - as separators in feed IDs.

If I remember correctly, it is because an _ is used to separate the region with the per-region feed ID to create the downloaded final file name.

Reply:
Should I just try replacing the underscore with a dash?

Reply:
Yeah, that should work!